pkg/cmd/pipeline/start.go

```diff
@@ -152,6 +153,7 @@ like cat,foo,bar
 	c.Flags().StringVarP(&opt.PrefixName, "prefix-name", "", "", "specify a prefix for the pipelinerun name (must be lowercase alphanumeric characters)")
 	c.Flags().StringVarP(&opt.TimeOut, "timeout", "", "1h", "timeout for pipelinerun")
 	c.Flags().StringVarP(&opt.Filename, "filename", "f", "", "local or remote file name containing a pipeline definition to start a pipelinerun")
+	c.Flags().BoolVarP(&opt.ParamUseDefaults, "params-use-defaults", "", false, "allow using deafult parameter values without prompting for input")
```

Comment:
For the flag name, maybe `--use-param-default` instead?

Also slight rewording for the description:

```suggestion
	c.Flags().BoolVarP(&opt.ParamUseDefaults, "use-param-defaults", "", false, "use default parameter values without prompting for input")
```

Reply:
Then I think we should also change the `ParamUseDefaults` field `startOptions` struct to `UseParamDefaults`. Let me update the PR with these changes.